### ISSUE TITLE
"Control the pointer using the keypad" does not work after restart.

### ISSUE THREAD

**Issue by Numnumberry** [linuxmint/cinnamon#9451]
```
 * Cinnamon version 4.6.6
 * Distribution - Mint 20
 * GeForce GTX 960
 * 64 bit
 ```
**Issue**
The option of "Control the pointer using the keypad" under Accessibility does not work after restarting the computer and waiting a few minutes after logging in.

**Steps to reproduce**
1) Turn the option on and verify that it works
2) Restart the computer
3) After logging in DO NOT MOVE THE MOUSE, wait for 2+ minutes
4) See that it does not work, the keypad reverts to default behavior'
5) Go into Accessibility and see that the option is still turned on
6) Turn the option off then back on and see that it now works

**Expected behaviour**
Controlling the pointer using the keypad is expected to work no matter how long the computer is at rest after logging in.

**Other information**
This issue was also in previous versions of Mint (like 19.3) but the bug was never fixed because the issue could not be reproduced, I believe now that waiting 2+ minutes will allow reliable reproduction. In 19.3 the option did not need to be toggled off then back on to work again, you only needed to open and close the Accessibility window to get it working again. I have tested after 1 minute, 1.5, 2, and 3 minutes and the issue only occurred after 2+ minutes. Using the keypad as mouse usually always works immediately after logging in.

**Comment by icarter09** [comment]
Cinnamon 4.8.6
Mint 20.1 64bit

I'm not able to reproduce this issue with the provided steps. @Numnumberry are you still experiencing this issue?

**Comment by Numnumberry** [comment]
I am still experiencing this issue

**Comment by mtwebster** [comment]
I can reproduce this on Mint 21 but I can't find a cause in Cinnamon - there is a timeout that can be set but it isn't enabled by default, and I confirmed it wasn't being explicitly applied.  It could be an xserver bug (or maybe feature?). I noticed that letting the machine idle for a few minutes will disable it - not just at login.

Either way, toggling numlock on and off again restores it.

**Comment by Numnumberry** [comment]
Sadly, I usually dont know that it turned itself off until I hit a number and it messes something up, but the numlock will at least be a bit quicker than opening Accessibility
"there is a timeout that can be set but it isn't enabled by default" - does this mean I can go about enabling a timeout (intending for it to be forever/never)? (: how could I do this?

**Comment by mtwebster** [comment]
You can try:
```
gsettings set org.cinnamon.desktop.a11y.keyboard disable-timeout 999999999
```
I tested it briefly and it seemed to work during the session, but it seemed to still timeout at startup.
